gtnet/model.py

```diff
@@ -2,33 +2,35 @@
 from pkg_resources import resource_filename
 import onnxruntime as rt
 import requests
+import pdb
 
 
-def _get_model_path(domain):
+def _get_model_path(model_type):
     """
-    Get the path to the ONNX model file for a given domain
-
+    Get the path to appropriate ONNX model files
     Parameters
     -----------
-    domain: {'bacteria', 'archaea'}
+    model_type: {'prediciton_model', 'confidence_model'}
```

Comment:
I'm not sure these values will resolve to the correct file path. Try using the manifest file to get the actual paths.

Reply:
wrapped up into config class now and this function/file is redundant